wgsl/index.bs

```diff
@@ -2866,7 +2893,7 @@ The size may include non-addressable padding at the end of the type.
 Consequently, loads and stores of a value might access fewer memory locations
 than the value's size.
 
-Alignment and size for host-shareable types are defined recursively in the
+Alignment and size of types are defined recursively in the
```

Comment:
Should this mention the fact that some types are excluded, e.g.handles, etc.
That's why the "host-shareable" qualification was there. (Did it get out of date?)

Reply:
It doesn't really matter since you can't touch that memory, but I could change this to non-opaque types.